docs/run/quantum-serverless.mdx

```diff
@@ -5,55 +5,79 @@ description: Run workloads with Quantum Serverless
 
 # Run workloads with Quantum Serverless
 
-Premium users can run their workloads remotely on classical compute made available through IBM Quantum Platform.
+Premium users can build, deploy, and run their workloads remotely on classical compute made available through the IBM Quantum Platform.
+
+Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) (note: these are accessible in the Premium Plan once you have logged into your IBM Quantum account) and explore more of the features of Quantum Serverless in the [documentation](https://qiskit-extensions.github.io/quantum-serverless/).
 
-For more information, see the [Quantum Serverless docs](https://qiskit-extensions.github.io/quantum-serverless/).
 
 <Admonition type="note">
     This is an experimental feature, subject to change.
 </Admonition>
 
-## Build a workflow
+## Qiskit Patterns with Quantum Serverless
+
+Creating utility-scale quantum applications generally requires a variety of compute resource requirements. You can use Quantum Serverless to easily submit quantum workflows for remote, managed execution. These quantum workflows can typically be implemented within a common pattern, called a Qiskit Pattern. A Qiskit Pattern is an intuitive, repeatable set of steps for implementing a quantum computing workflow.
+
+Steps in a Qiskit Pattern:
+
+1. Map classical inputs to a quantum problem
```

Comment:
step 1 is now "Generate quantum circuits and operators." (generate instead of map)